root/layout/components/sidebar/RemoveLink.js

```diff
@@ -12,6 +12,7 @@ import * as React from 'react';
 import entityHref from '../../../static/scripts/common/utility/entityHref';
 
 type Props = {
+  +$c: CatalystContextT,
```

Comment:
FWIW since hooks were added, it's usually easier to use https://reactjs.org/docs/hooks-reference.html#usecontext than pass `$c` everywhere.